project/src/js/ui/tournament/detail/socketHandler.js

```diff
@@ -0,0 +1,23 @@
+import * as xhr from '../tournamentXhr';
```

Comment:
Ok, removed socketHandler file and the resync handler and just put the remaining handlers in the tournamentDetailCtrl.js file